### PR TITLE
docs: rework Ubuntu instructions

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

**Review by alexsavulescu — DISMISSED** [review]
SGTM

**Comment by pramodk** [comment]
I `think` there might be some additional minor changes required to get STEPS people happy i.e. able to build STEPS painlessly (just because some issues in packages like `med`). Lets see @cattabiani's feedback.

**Comment by cattabiani** [comment]
I still have some issues with med and gmsh. I will add my changes on top of this if I need to

File: bluebrain/documentation/setup_personal.md [hunk 1]
```diff
@@ -45,31 +52,29 @@ by using Spack's environment feature.
...
-    $ . spack/share/spack/setup-env.sh
-    $ spack compiler find
+    $ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+    $ sudo apt update
+    $ sudo apt install build-essential gcc-11 g++-11 gfortran-11
 
-### Ubuntu 20.04
+Then clone and configure Spack:
 
     $ git clone -c feature.manyFiles=true https://github.com/BlueBrain/spack.git
```
Comment (cattabiani) [inline bluebrain/documentation/setup_personal.md]:
```suggestion
    $ git clone -c feature.manyFiles=true git@github.com:BlueBrain/spack.git
```

Comment (cattabiani) [reply bluebrain/documentation/setup_personal.md]: In this way it does not complain about keys...

Comment (pramodk) [reply bluebrain/documentation/setup_personal.md]: I don't know which keys but this should be `https` as repo is public.

Comment (cattabiani) [reply bluebrain/documentation/setup_personal.md]: I am talking about the fact that github stopped accepting simple name/password and requires a key to commit

Comment (cattabiani) [reply bluebrain/documentation/setup_personal.md]: can't you use git:*** if the repo is public?

Comment (matz-e) [reply bluebrain/documentation/setup_personal.md]: I just use `gh repo clone BlueBrain/spack`, and it sets up `https` by default, IIRC

File: bluebrain/documentation/setup_personal.md [hunk 2]
```diff
@@ -25,56 +34,70 @@ conjunction with Apple's CLang:
...
+    (providing /usr/bin/python): /usr/bin/python3.8. Nothing to configure.
+
+## Spack Setup
+
 Now clone our version of Spack and find compilers and external packages:
 
     $ git clone -c feature.manyFiles=true https://github.com/BlueBrain/spack.git
     $ . spack/share/spack/setup-env.sh
     $ spack compiler find
     $ spack external find
```
Comment (cattabiani) [inline bluebrain/documentation/setup_personal.md]: Still checking stuff but I think that the steps setup should go into steps and not in the general spack

Comment (matz-e) [reply bluebrain/documentation/setup_personal.md]: Sure

File: bluebrain/documentation/setup_personal.md [hunk 3]
```diff
@@ -25,56 +34,70 @@ conjunction with Apple's CLang:
...
-Use Spack on the workstations provided by the project.
+#### When Building Steps
 
-### Ubuntu 18.04
+Use the following commands to skip building the FLTK GUI interface for
+GMSH, as it pulls in too many dependencies:
 
-We build Docker images based on Ubuntu 18.04, and the same settings can be
-used to set Spack up on the desktops:
+    $ spack config add packages:gmsh:variants:"~mmg~fltk"
```
Comment (cattabiani) [inline bluebrain/documentation/setup_personal.md]: I am adding a doc in steps. This is not needed

**Review by cattabiani — CHANGES_REQUESTED** [review]
Just a small note. The rest passed the tests!

File: bluebrain/documentation/setup_personal.md (hunk 3 above)
Comment (cattabiani) [reply bluebrain/documentation/setup_personal.md]: I do not understand why I cannot give a suggestion here

Comment (matz-e) [reply bluebrain/documentation/setup_personal.md]: You should be able to!

I can take this out.

Comment (matz-e) [reply bluebrain/documentation/setup_personal.md]: Done. Thank you for testing and reviewing!

**Review by cattabiani — APPROVED** [review]
LGTM!

**Pull request merged**